README.md

```diff
@@ -151,6 +151,14 @@ Tests should always eventually be run within a docker container to ensure a cons
 
 Prefix any of the `test-` targets with `docker-` to run them in a container eg: `make docker-test-server`.
 
+### Network issues diagnose
+
+It is possible to diagnose Network issues related with lost documents between APM Agent, APM server, or Elasticsearch,
```

Comment:
This doc content is good but should we broken up into one (or more lines) per sentence.  Markdown will render together into a paragraph as long as there are no blank lines in between sentences.

Reply:
I've broken the lines near the 80 characters.